.travis.yml

```diff
@@ -11,13 +11,13 @@ python:
 install:
   - "pip install ."
   - "pip install -r test_requirements.txt"
-  - "pip install flake8"
-before_script: "flake8 --max-complexity 10 hyperframe test"
+  - "pip install flake8 || true"
+before_script: "if [ -x "$(command -v flake8)" ] ; then flake8 --max-complexity 10 hyperframe test ; fi"
```

Comment:
Aha, I think I see the issue. The quoting here is probably throwing the YAML parser off.

If you look at https://travis-ci.org/python-hyper/hyperframe/requests, the error it's reporting is "Could not parse .travis.yml".

Reply:
ah right - thanks! I fixed it.